ouroboros-network-framework/test/Test/Simulation/Network/Snocket.hs

```diff
@@ -338,30 +354,299 @@ toBearerInfo abi =
 -- Properties
 --
 
+data TestError = UnexpectedOutcome
+  deriving (Show, Eq)
+
+instance Exception TestError
```

Comment:
If I am not mistaken we're never `throwIO` these errors, so the `Exception` instance is not really needed.  I put a comment below how to avoid `toExceptions` as well.

Reply:
fixed!